spring-cloud-task-docs/src/main/asciidoc/preface.adoc

```diff
@@ -36,9 +36,9 @@ If you are just getting started with Spring Cloud Task or with 'Spring' in gener
 suggesting reading the <<getting-started.adoc#getting-started>> chapter.
 
 To get started from scratch, read the following sections:
-* "`<<getting-started.adoc#getting-started-introducing-spring-cloud-task>>`"
-* "`<<getting-started.adoc#getting-started-system-requirements>>`"
+<<getting-started.adoc#getting-started-introducing-spring-cloud-task, Introducing Spring Cloud Task>>
+<<getting-started.adoc#getting-started-system-requirements, System Requirements>> +
```

Comment:
Need to add a line separator so that they will appear on on seperate lines.   Will fix on merge.